tests/scripts/run_performance.sh

```diff
@@ -36,12 +36,12 @@ run_perf_models_llm_javelin() {
     local tt_arch=$1
     local test_marker=$2
 
-    env pytest models/demos/falcon7b/tests -m $test_marker
-
```

Comment:
Why was this moved to later in the pipeline?

Reply:
For testing purposes, I am seeing some hangs after our tests